modules/busco/main.nf

```diff
@@ -0,0 +1,82 @@
+process BUSCO {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::busco=5.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/busco:5.3.2--pyhdfd78af_0':
+        'quay.io/biocontainers/busco:5.3.2--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta, stageAs: 'tmp_input/*')  // Required:    meta map, and fasta sequence files
+    each lineage                                          // Required:    lineage to check against
+    path busco_lineages_path                              // Recommended: path to busco lineages - downloads if not set
+    path config_file                                      // Optional:    busco configuration file
+
+    output:
+    tuple val(meta), path("*-busco"), emit: busco_dir
+    path "versions.yml"             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}-${lineage}"
+    def busco_config = config_file ? "--config $config_file" : ''
+    def busco_lineage_dir = busco_lineages_path ? "--download_path ${busco_lineages_path}" : ''
+    """
+    # Nextflow changes the container --entrypoint to /bin/bash (container default entrypoint: /usr/local/env-execute)
+    # Check for container variable initialisation script and source it.
+    if [ -f "/usr/local/env-activate.sh" ]; then
+        # . "/usr/local/env-activate.sh"  # Errors out because of various unbound variables
+        export PATH='/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
+        export CONDA_PREFIX='/usr/local'
+        export CONDA_SHLVL='1'
+        export CONDA_DEFAULT_ENV='/usr/local'
+        export CONDA_PROMPT_MODIFIER=''
+        . "/usr/local/etc/conda/activate.d/activate-r-base.sh"
+        . "/usr/local/etc/conda/activate.d/augustus.sh"
+        . "/usr/local/etc/conda/activate.d/openjdk_activate.sh"
```

Comment:
Would this work ? Less maintenance effort
```suggestion
        set +u  # Otherwise, errors out because of various unbound variables
        . "/usr/local/env-activate.sh"
        set -u
```